src/wp-includes/class-wp-theme-json.php

```diff
@@ -2053,6 +2053,22 @@ protected static function compute_style_properties( $styles, $settings = array()
 			return $declarations;
 		}
 
+		$can_use_cached = ! wp_is_development_mode( 'theme' );
+		if ( $can_use_cached ) {
+			$func_args_hash = md5( wp_json_encode( func_get_args() ) ); // phpcs:ignore PHPCompatibility.FunctionUse.ArgumentFunctionsReportCurrentValue.Changed
```

Comment:
I did a small experiment interchanging `wp_json_encode` with `serialize`.
On average `serialize` was `15%` faster than `wp_json_encode`. There were a few random outliers in 85 runs where  `wp_json_encode` performed better. 

However, the string size produced by `serialize` was almost 2x the size of `wp_json_encode`. 

The cost of `md5` increases with an increase in string size. 
On this line both `seralize+md5` and `wp_json_encode+md5` had near similar cost, but in `wp_get_global_settings_hash`  the cost of `wp_json_encode+md5` was lower compared to the counterpart.